testing/test_modules.out

```diff
@@ -5,14 +5,47 @@ RESULT skip base.startup.dhcp Using static ip 10.20.0.5
 RESULT skip base.switch.ping No local IP has been set, check ext_loip in system.conf
 RESULT pass base.target.ping target reached %% 10.20.0.5
 Testing tls alt
-RESULT skip security.tls.v3
-RESULT skip security.x509
+RESULT skip security.tls.v1
+RESULT fail security.tls.v1.x509
+RESULT skip security.tls.v1_2
+RESULT fail security.tls.v1_2.x509
+RESULT skip security.tls.v1_3
+RESULT fail security.tls.v1_3.x509
+RESULT fail security.tls.v1.x509
+RESULT skip security.tls.v1_2
```

Comment:
results are being duplicated still?  This is weird because when I run test_modules.sh locally I don't see the duplication.  Anyway... I'll investigate further but I wouldn't consider it a blocker for your PR since I don't think it's the tls-test code!

Reply:
Not sure what duplication you're seeing, this looks just like the 3 skip/pass/fail scenarios. I still don't really know how the faux devices are built.  There probably needs to be some verification that TLS 1.3 is actually available in them separate to the test to rule out which one is the issue.